### ISSUE TITLE
Composite Injector does not function properly with certain equipments

### ISSUE THREAD

**Issue by proboscismonky** [TeamMoonstorm/Starstorm2#624]
Royal Capacitor, Tricorn, and Woodsprite UI elements do not appear at all when in main slot or offhand
Woodsprite passive does not work in offhand slot
Recycler UI graphic does not appear at all when in main slot, some parts of the graphic are missing when in offhand
Credit card does not work in the offhand slot

Crowdfunder has some potential issues, but I'm not really sure what the intended behavior is supposed to be. Below is how it works now
No offhand equipments are usable when crowdfunder is in the main slot
Using equipment in main hand does not toggle crowdfunder in the offhand

**Comment by alicealysia** [comment]
Yeah, this might be the single most broken and useless legendary in the game
